src/initializeCacheClient.js

```diff
@@ -0,0 +1,35 @@
+/* Memcached cache client. */
+import { get } from 'lodash';
+import { bind, getContainer, getMetadata } from '@globality/nodule-config';
+
+import Cache from './memcached-promisify';
+
+function createCacheClient() {
+    const { config } = getContainer();
+    const memcachedConfig = get(config, 'cache.memcached');
+    const cacheHost = get(config, 'cache.hosts', '').split(',');
```

Comment:
Do we really have to `.split(',')`, Maybe `memcached` can handle strings

Reply:
it can't. see https://github.com/3rd-Eden/memcached#options